### PR TITLE
Welp... here goes nothing!

### PR DESCRIPTION
**Pull request by thisandagain**
First pull request includes basic functionality for filtering & search (GH-1) as well as some minor refactoring and visual updates (essentially just prep for an actual design pass). Next up I was going to tackle GH-2 and a UX/design pass.

Thanks for starting this project! This is something that I've wished existed for quite a while. :smile:

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+     * Generates query object for the Google Fusion Tables API.
+     *
+     * @return {Object}
+     */
+    function generateQuery () {
+        var search  = generateSearchQuery($('#search').find('input').val()),
+            filter  = generateFilterQuery(),
+            and     = (search !== '' && filter !== '') ? ' AND ' : '';
+
+        console.dir(search + and + filter);
```
Comment (bobthecow) [inline js/index.js]: :warning:

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+    /**
+     * Generates a search query.
+     *
+     * @param {String} Search term
+     *
+     * @return {String}
+     */
+    function generateSearchQuery (term) {
+        var q  = term,
```
Comment (bobthecow) [inline js/index.js]:
``` js
var q = term.replace(/^\s+|\s+$/g, "").replace(/'/g, '\\\'').toLowerCase();
    qp = "'Resource Name' CONTAINS IGNORING CASE '",
    qs = "'";

if (q === '') return '';
return qp + q + qs;
```

Prevent SQL injection (at least naïvely) and as a bonus, trim so the `if (q === '')` works.

**Comment by bobthecow** [comment]
Other than the issues mentioned it looks great. Thanks for your work so far.

We can leave the issue link as GitHub for now. If we decide to revert it's not the end of the world.

Clean up that SQL injection and debugging and I'll hit the merge button :)

File: js/index.js (hunk 2 above)
Comment (thisandagain) [inline js/index.js]: Good catch. Late night sloppy hacks are sloppy. :cat2:

Comment (thisandagain) [inline js/index.js]: On a related note... I really need to dig into the Fusion Table API docs. I assumed this was read-only access.

**Comment by thisandagain** [comment]
All set. Are you in SF/EB? I was thinking about hacking from Sudo Room this evening on the design pass if you are around.

File: js/index.js (hunk 2 above)
Comment (bobthecow) [inline js/index.js]: I honestly don't know. I was more worried about unintentional issues. Users not being able to figure out why they can't search for `Earl's Electronics Emporium` or something.

**Pull request merged**